src/assessment.handlebars

```diff
@@ -1046,8 +1046,9 @@
   </div>
   <div class="slide main-menu" id="main-menu">
     <div class="flow">
-      <div class="center donation-banner">Text SEAP02 £2 to 70070 to help us support more people like you.
-          <img src="images/jtg_logo.png" alt="Just Text Giving logo"/>
+      <div class="donation-banner">
+        <p class="center">If you find this website useful, please consider donating.</p>
+        <p class="center">Text <b>'SEAP02 £2'</b> to <b>70070</b> to help us support more people like you.</p>
```

Comment:
I think we should use `&#x2018;` and `&#x2019;` to be consistent with the existing encoding in these apps.

Reply:
It doesn't seem massively consistent use of `&#39;` for single quote too - which should be apostrophe.

But we might as well use the correct open and close single encode for this update.